jekyll/_cci2/env-vars.md

```diff
@@ -49,6 +49,47 @@ Running scripts within configuration
 may expose secret environment variables.
 See the [Using Shell Scripts]({{ site.baseurl }}/2.0/using-shell-scripts/#shell-script-best-practices) document for best practices for secure scripts.
 
+
+### What Environment Variables Look Like In An Example Config
```

Comment:
-> Example Configuration of Environment Variables (this will perform better in SEO and is easier to translate/proper english)